packages/pf4-component-mapper/src/text-field/text-field.js

```diff
@@ -26,9 +26,8 @@ const TextField = (props) => {
         {...rest}
         id={id || input.name}
         isRequired={isRequired}
-        isReadOnly={isReadOnly}
         isDisabled={isDisabled}
-        readOnly={isReadOnly}
+        readOnlyVariant={isReadOnly ? 'default' : undefined}
```

Comment:
While `readOnly` was still working - it is no longer documented (use `readOnlyVariant` instead): https://www.patternfly.org/components/forms/text-input/#props